WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -209,3 +284,7 @@ private extension ProductDetailPreviewViewModel {
                                                             comment: "Format of all 3 dimensions on the Shipping Settings row - L x W x H[unit]")
     }
 }
+
+private enum IdentifyLanguageError: Error {
```

Comment:
Nit: Should we keep this error internal instead of duplicating it in different features?

Reply:
I agree. Done in 94d5e9b2b1e8b1b1d61baf11c64f6dce932e5579